scripts/prepare_automatic_backups.py

```diff
@@ -58,10 +58,10 @@ def update_cron_dict(cron_dict):
 
     Args:
         cron_dict: dict(str ,str). Content of yaml file in dictionary type.
-            'description' : 'weekly bakcup'
-            'url' : 'new backup url'.
-            'schedule' : 'sheduled time of cron job'.
-            'target' : 'cloud storage name'
+            'description': str. {{ weekly backup.}}
+            'url': str. {{new backup url.}}
+            'schedule': str. {{scheduled time of cron job.}}
+            'target': str. {{cloud storage name.}}
```

Comment:
Please remove all the curly braces "{" and "}". That were just used as a placeholder that you need to fill in.

Reply:
resolved it